client/components/NavBar/HomeIcon/HomeIcon.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
```

Comment:
I'm not sure if each item will need it's own component & no styles are currently being used so we don't need to import that as well.

Reply:
Commented out style import. Will delete later after we add style to the overall app and those lines remain unnecessary. Left each icon as its own component, but will consolidate that too, if it turns out to serve no purpose.